app/Http/Requests/StoreOrganisationMember.php

```diff
@@ -28,7 +28,9 @@ public function rules()
     {
         return $this->clean([
             'organisation_id' => 'required|exists:organisations,id',
-            'character_id' => 'required|exists:characters,id',
+            'characters' => 'required_without:character_id|array|min:1',
```

Comment:
Create a new request called StoreOrganisationMembers instead of making a complicated one that could potentially break the API